ZXing.Net.MAUI/Apple/CameraManager.apple.cs

```diff
@@ -106,8 +106,8 @@ public void UpdateCamera()
 					captureDevice = null;
 				}
 
-				var devices = AVCaptureDevice.DevicesWithMediaType(AVMediaType.Video);
-				foreach (var device in devices)
+                var devices = AVCaptureDevice.DevicesWithMediaType(AVMediaTypes.Video.ToString());
```

Comment:
I'm not sure about `ToString`, but we don't have an overload that allows the `AVMediaTypes` enum.